shuup_tests/importer/test_product_import.py

```diff
@@ -68,6 +68,8 @@ def test_sample_import_no_match():
     tax_class = get_default_tax_class()
     product_type = get_default_product_type()
     supplier = get_default_supplier()
+    supplier.stock_managed = True
+    supplier.save()
```

Comment:
Are you changing this test because it doesn't pass without the change any more? I guess you're just trying to make it test the import with stock management enabled. It would be better to parameterize this (do_stocks = True and do_stocks = False), since that would prove it still works without stock management too.